### PR TITLE
docs: update 1.8 release notes for scst-store

### PR DESCRIPTION
**Pull request by julia-pu**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

File: release-notes.hbs.md [hunk 1]
```diff
@@ -89,7 +89,9 @@ The following issues, listed by component and area, are resolved in this release
 
 #### <a id='1-8-0-COMPONENT-NAME-ri'></a> v1.8.0 Resolved issues: COMPONENT-NAME
 
-- Resolved issue description.
+#### <a id='1-8-0-scst-store-ri'></a> v1.8.0 Resolved issues: Supply Chain Security Tools - Store
+
+- the expired certificate issue where users need to restart the metadata-store pods when the internal db certificate gets rotated by cert-manager is resolved. Users should no longer see this issue with the default internal db, but the solution does not cover the case of an external db.
```
Comment (ohlookadollar) [inline release-notes.hbs.md]: Do we explain anywhere that we remove tls to the internal db and internal encryption should be handled on the internal network layer?

Comment (julia-pu) [reply release-notes.hbs.md]: I wasn't sure if we want to explain what we did to fix it, if that's too much information

Comment (julia-pu) [reply release-notes.hbs.md]: Should I add that?

Comment (Manifaust) [inline release-notes.hbs.md]: Can you capitalize the first sentence

Comment (Manifaust) [reply release-notes.hbs.md]: change db to database

Comment (Manifaust) [reply release-notes.hbs.md]: Start the sentence with "This release fixes the issue with expired certificates where...". Then you can remove the word "resolved" at the end when you do this refactor.

**Review by Manifaust — APPROVED** [review]

**Review by ohlookadollar — APPROVED** [review]

File: release-notes.hbs.md (hunk 1 above)
Comment (julia-pu) [reply release-notes.hbs.md]: okay made those changes!

**Pull request merged**